atc/lidar/scanner.go

```diff
@@ -53,6 +58,11 @@ func (s *scanner) scanResources(ctx context.Context, resources []db.Resource, re
 	logger := lagerctx.FromContext(ctx)
 	waitGroup := new(sync.WaitGroup)
 	for _, resource := range resources {
+		if resource.BuildSummary() != nil && resource.BuildSummary().Status == atc.StatusStarted && resource.BuildSummary().StartTime + 60 > time.Now().Unix() {
```

Comment:
Do we need this here? Currently, we have [logic in `CheckFactory`](https://github.com/concourse/concourse/blob/0874fc5969abc560af526b4c812147871654e154/atc/db/check_factory.go#L105-L116) to check if the interval has passed

Reply:
The logic in`CheckFactory.TryCreateCheck` is to check last check time. Here, when a slow check is still running, I want to avoid creating duplicate checks. But I agree, this logic is better to be added in `CheckFactory`.